profile/README.md

```diff
@@ -0,0 +1,19 @@
+Welcome to the Scala Center organization!
+
+Our primary focus is to:
+
+- Guide and support the Scala community
+- Coordinate and develop open source libraries and tools for the benefit of all
+    Scala users
+- Provide deep and quality educational materials for Scala
+
+You can read more about our purpose, see what we're working on, and find records
+of our doings on [scala.epfl.ch](https://scala.epfl.ch/).
+
+| Our communication channels                                                                 |
+| -------------------------------------------------------------------------------------------|
+| [Twitter](https://twitter.com/scala): for general announcements about Scala                |
+| [Discord](https://discord.gg/q8VJD4tCK4): to ask questions and discuss Scala-related topics|
+| [LinkedIn](https://www.linkedin.com/company/scala-center/): Scala Center happenings        |
```

Comment:
I think we could also add the following channels:

~~~ text
| [Contributors forum](https://contributors.scala-lang.org/c/scala-center/25): project roadmaps and updates |
| [Scala website](https://scala-lang.org/blog): general announcements about Scala |
~~~

Reply:
Good idea, added.